src/setup_db.ts

```diff
@@ -0,0 +1,50 @@
+import express from "express";
+import db from "./db.ts";
+import "dotenv/config";
+
+export async function setupDB() {
+    // Setup express server
+    const app = express();
+    app.use(express.json());
+    app.get("/", (req, res) => {
+        return res.json("Established connection!");
+    });
```

Comment:
This should happen after the DB is correctly connected and setup. Should maybe be moved to the main ts file or a routes ts file?